charts/cluster-shield/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: cluster-shield
 description: Cluster Shield Helm Chart for Kubernetes
 type: application
-version: 1.3.1
-appVersion: "1.3.1"
+version: 1.3.2
+appVersion: "1.3.2"
```

We are not releasing a new version fo the _Cluster Shield_ but only in the chart so this should not be changed.
```suggestion
appVersion: "1.3.1"
```